inst/help/ttestonesample.md

```diff
@@ -1,95 +1,94 @@
  - Exclude cases listwise: In case of multiple t-tests within a single analysis, each t-test will be conducted using only cases with valid data for all dependent variables. Sample size is therefore constant across the tests.
 
 ### Output
 -------
 
-#### One Sample T-Test 
+#### One Sample T-Test
 - The first column contains the variables for which the analysis is performed.
-- Test: The type of test that is selected. If only one test is selected, this column will not be displayed. In this scenario, the table only displays the results for the selected test. 
-- Statistic: The test statistic. For the Student's t-test this is the value of the t-statistic. For the Wilcoxon signed-rank test this is the value of the W-statistic. For the Z test this is the value of the Z-statistic. 
+- Test: The type of test that is selected. If only one test is selected, this column will not be displayed. In this scenario, the table only displays the results for the selected test.
+- Statistic: The test statistic. For the Student's t-test this is the value of the t-statistic. For the Wilcoxon signed-rank test this is the value of the W-statistic. For the Z test this is the value of the Z-statistic.
 - df: Degrees of freedom.
 - p: The p-value.
-- Mean difference: Average difference between the data points and the test value. This column is only named 'Mean difference' when the test `Student` is selected. When the test `Wilcoxon signed-rank` and/or the Z test is selected, this column is called 'Location parameter'. 
-- Location parameter: For the Student's t-test and the Z test, the location parameter is given by mean difference; for the Wilcoxon signed-rank test, the location parameter is given by the Hodges-Lehmann estimate. This column is only called 'Location parameter' when `Wilcoxon signed-rank` test and/or the Z test is selected, otherwise this column is called 'Mean difference'. 
-- SE Difference: Standard error of the mean of the difference scores. 
-- % CI for Mean difference/location parameter: The confidence interval for the mean difference/location parameter of the difference scores. By default this is set to 95%. 
-  - Lower: The lower bound of the confidence interval. 
-  - Upper: The upper bound of the confidence interval. 
-- Effect Size: For the Student t-test, the effect size is given by Cohen's d; for the Wilcoxon test, the effect size is given by the matched rank biserial correlation; for the Z test, the effect size is given by Cohen's d (based on the provided population standard deviation). 
-- % CI for Effect Size: The confidence interval for effect size. By default this is set to 95%. 
-    - Lower: The lower bound of the confidence interval. 
+- Sample Mean: The observed average of the data points. This column is only named 'Sample Mean' when the test `Student` or `Z Test` is selected. When the test `Wilcoxon signed-rank` and/or the Z test is selected, this column is called 'Location Estimate'.
```

Comment:
Here the old explanation is simply more correct. Rather than 
```
Average difference between the data points and the test value ...
```
this could also be
```
Sample mean minus the test value
```